core/storage/classifier/gae_models.py

```diff
@@ -181,6 +181,25 @@ def create(
         training_job_instance.put()
         return instance_id
 
+    @classmethod
+    def query_training_jobs(cls, inlcude_deleted=False):
```

Comment:
See my new comment in latest review. If you go with that method you also get to know whether more models are available or not. You can add extra condition at the end that if `more` is False then break the loop since we don't have a job.